src/index.ts

```diff
@@ -56,6 +57,45 @@ passport.use('local', new LocalStrategy(
   }
 ))
 
+if ( process.env.GOOGLE_CLIENT_ID && process.env.GOOGLE_CLIENT_SECRET ) {
```

Comment:
This should be moved to src/config/index.ts if possible

Reply:
Done as part of https://github.com/Wingysam/Christmas-Community/pull/138/commits/173add9093dd9ad4ab64fbeb9751ead78477d44e to make the Google strategy async.